src/routes/settings/+page.svelte

```diff
@@ -0,0 +1,320 @@
+<script lang="ts">
+    import type { PageData } from './$types';
+    import type { TranslationPair } from '$lib/types/base';
+    import type { column } from '$lib/types/table';
+    import { patchToApi, postToApi, deleteToApi } from '$lib/utils/http-service';
+    import { debounce } from '$lib/utils/debounce';
+    import { createSettingsTableSorter, SortName } from './settings-table-sorter';
+    import { log } from '$lib/logger';
+    import { isApiErrorWithMessage } from '$lib/utils/http-errors';
+    import { searchParameters, ssp } from '$lib/utils/sveltekit-search-params';
+    import Table from '$lib/components/Table.svelte';
+    import Select from '$lib/components/Select.svelte';
+    import TrashIcon from '$lib/icons/TrashIcon.svelte';
+    import Modal from '$lib/components/Modal.svelte';
+
+    interface TranslationPairsLanguage {
+        languageId: number;
+        englishDisplay: string;
+    }
+
+    interface NewTranslationPairResponse {
+        id: number;
+    }
+
+    export let data: PageData;
+
+    let isTransacting = false;
+    let search = '';
+    let translationPairs: TranslationPair[] = [];
+    let filteredTranslationPairs: TranslationPair[] = [];
+    let translationPairsLanguages: TranslationPairsLanguage[] = [];
+    let openDeleteModal = false;
+    let openCreateModal = false;
+    let newKey = '';
+    let newValue = '';
+    let currentLanguageDisplayname = '';
+    let currentDeleteTranslationPairId = 0;
+    let errorMessage = '';
+    let openErrorModal = false;
+
+    // eslint-disable-next-line
+    let table: Table<any> | null;
```

Comment:
I think we can type this as `Table<TranslationPair> | undefined` and then the eslint disable isn't needed. I switched other spots we had `Table<any>` in one of my recent PRs.